client/account-recovery/reset-password/reset-password-email-form/index.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+
+const ResetPasswordEmailSent = ( props ) => {
+	const {
+		email,
+		translate,
+	} = props;
+
+	return (
+		<Card>
+			<h2 className="reset-password-email-form__title">
+				{ translate( 'Reset your password' ) }
+			</h2>
+			<p>
+				{ translate(
+					"We've sent an email to {{strong}}%(email)s{{/strong}} with a link to complete your password reset.",
+					{
+						components: { strong: <strong /> },
+						args: { email }
+					}
+				) }
+			</p>
+			<p>
+				{ translate( "If you haven't received it, please check your spam or junk folder." ) }
+			</p>
+			<a href="#">{ translate( "You didn't receive it?" ) }</a>
```

Comment:
Actually I think this is a good suggestion, especially given the fact that in its sibling SMS form we are saying "No SMS?". Updated by 3498e3c.